hs_core/search_indexes.py

```diff
@@ -471,10 +471,17 @@ def prepare_start_date(self, obj):
                     clean_date = coverage.value["start"][:10]
                     if "/" in clean_date:
                         parsed_date = clean_date.split("/")
-                        start_date = parsed_date[2] + '-' + parsed_date[0] + '-' + parsed_date[1]
-                    else:
+                        if len(parsed_date) == 3:
```

Comment:
So this block here is the fix for the dates problem?  Everything above this looks like small tweaks and optimizations.

Reply:
Yes. It doesn't accept partial dates. Previously it crashed ingestion and the resource did not appear.